tests/integration/test_caching.py

```diff
@@ -190,9 +190,8 @@ def sleep_for_a_bit():
 
 def get_all_ids_from_Project():
     """Fetches all currently available Synapse IDs from the parent Project."""
-    
-    others = syn.chunkedQuery('select id from entity where parentId=="%s"' % syn.test_parent.id)
-    return [result['entity.id'] for result in others]
+    others = list(syn.getChildren(syn.test_parent.id))
+    return [result['id'] for result in others]
```

Comment:
you can avoid generating the list of children. just use

`for result in syn.getchildren(...)`